client/components/TestRun/index.jsx

```diff
@@ -68,21 +68,26 @@ const TestRun = () => {
 
     const { runId: testPlanRunId, testPlanReportId } = params;
 
-    const { loading, data, error } = useQuery(
+    const { loading, data, error, startPolling, stopPolling } = useQuery(
```

Comment:
I wonder if we want to announce new test results coming in via screenreader. I know that isn't in the spec but VoiceOver is very quiet during these updates.

Reply:
I wonder if setting up the overall status section (upper right) as a live region when the job is running might be nice - but I don't necessarily like to assume I know what the screen reader user would want in this situation.  Happy to bring this up and maybe we push the screen reader update to it in the next issue?

Reply:
Sounds good!